packages/block-editor/src/components/image-editor/use-save-image.js

```diff
@@ -27,7 +24,7 @@ export default function useSaveImage( {
 	const cancel = useCallback( () => {
 		setIsInProgress( false );
 		onFinishEditing();
-	}, [ setIsInProgress, onFinishEditing ] );
```

Comment:
The `setIsInProgress` is a state `set` function; it could be omitted from dependencies.